ra_aid/agent_utils.py

```
@@ -149,69 +286,82 @@ def run_research_agent(
         code_snippets=code_snippets,
-        related_files=related_files
+        related_files=related_files,
+        project_info=formatted_project_info,
     )
 
-    # Set up configuration
-    run_config = {
-         "configurable": {"thread_id": thread_id},
-        "recursion_limit": 100
-    }
+    config = _global_memory.get("config", {}) if not config else config
```

Ruff (0.8.2)

Use `config if config else _global_memory.get("config", {})` instead of `_global_memory.get("config", {}) if not config else config`

Replace with `config if config else _global_memory.get("config", {})`

(SIM212)